### PR TITLE
Add file upload field to atributes

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds file attribute upload field to attributes component.

~~:warning: This PR contains changes from https://github.com/mirumee/saleor-dashboard/pull/832, which should be merged first.~~

Feature branch: https://github.com/mirumee/saleor-dashboard/pull/884

**PR intended to be tested with API branch:**  feature/file-uploads
https://github.com/mirumee/saleor/pull/6568

### Screenshots

Zrzut ekranu 2020-12-3 o 17 27 27
Zrzut ekranu 2020-12-3 o 17 27 32

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://feature-file-uploads.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1841 Add file upload field to attributes component](https://app.clickup.com/t/2549495/SALEOR-1841)

**Comment by orzechdev** [comment]
Cypress fails due to known issue with not using API_URI set in PR. Locally tests pass.

File: src/components/FileUploadField/FileUploadField.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    uploadFileContent: {
+      color: theme.palette.primary.main,
+      fontSize: "1rem"
+    }
+  }),
+  { name: "FileUploadField" }
+);
+
+const FileUploadField: React.FC<FileUploadFieldProps> = props => {
+  const { disabled, fileName, onFileUpload, onFileDelete, ...rest } = props;
```
Comment (krzysztofwolski) [inline src/components/FileUploadField/FileUploadField.tsx:34]: Maybe input props should be separate prop?

Comment (orzechdev) [reply src/components/FileUploadField/FileUploadField.tsx:34]: sure, good idea

Comment (orzechdev) [reply src/components/FileUploadField/FileUploadField.tsx]: updated

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**